src/onegov/town6/views/chat.py

```diff
@@ -117,7 +117,9 @@ def view_chats_archive(self, request):
     form=ChatInitiationForm)
 def view_chat_form(self: ChatCollection, request, form):
 
-    if not request.app.chat_active and not request.is_manager:
+    if not request.app.chat_open(
+        request
+    ) and not request.is_manager:
```

Comment:
This fits on one line
```suggestion
if not request.app.chat_active and not request.is_manager:
```